aws-ssm-document/aws-ssm-document.json

```diff
@@ -181,17 +150,10 @@
             "minItems": 1
         },
         "DocumentVersion": {
-            "description": "The version of the document that you want to update.",
+            "description": "The latest version number of document",
```

Comment:
Since we only allow updating on the latest version, shall we remove $DEFAULT from the allowed pattern?

Reply:
yeah, good point. It's a readOnly property actually, so it's not set by the user but will return the latest version number of the document. I'll remove it and send another revision.